react/src/components/masthead/components/SprkMastheadSelector/SprkMastheadSelector.js

```diff
@@ -12,7 +12,7 @@ class SprkMastheadSelector extends Component {
     this.state = {
       isOpen: false,
       triggerText: props.defaultTriggerText,
-      choiceItems: props.choices.items.map(item => ({
+      choiceItems: props.choices.items.map((item) => ({
```

Comment:
Verify if we added this to the spark exports @RVMendoza

Reply:
i got this! i added it. I want it added so that teams can use spark components to build custom components.